src/components/new-request/name-check/name-check-conflicts.vue

```diff
@@ -0,0 +1,320 @@
+<template>
+  <v-data-table :headers="headers"
+                fixed
+                hide-default-header hide-default-footer
+                :items="items"
+                item-key="problem"
+                single-expand
+                :expanded.sync="expanded"
+                no-data-text=""
+                show-expand>
+    <template v-slot:item="{ item, headers, expand, isExpanded }">
+      <tr v-if="item.info === ITEM_TYPES.NO_ISSUES" class="no-hover">
+        <v-row class="mb-5 pl-4 py-5 border-top border-bottom" justify="center" no-gutters>
+          <v-col cols="auto">
+            <b class="copy-bold">{{ item.problem }}</b>
+          </v-col>
+        </v-row>
+      </tr>
+      <tr v-else-if="item.info === ITEM_TYPES.LOADING" class="no-hover">
+        <v-row class="pl-4 py-5 border-top" no-gutters>
+          <v-col cols="auto">
+            <v-skeleton-loader type="button"/>
+          </v-col>
+        </v-row>
+      </tr>
+      <tr v-else-if="item.info && item.info.includes('error')" class="no-hover">
+        <v-row v-if="item.info && item.info.includes('error')" class="conflict-row py-5 px-4 border-top" no-gutters>
+          <v-col class="nudge-down" cols="auto">
+            <v-icon :color="item.iconColor">{{ item.icon }}</v-icon>
+          </v-col>
+          <v-col class="table-text pl-3 pt-1" cols="8">
+            <span v-html="item.problem"/>
+          </v-col>
+          <v-col class="pl-5" cols="auto">
+            <v-btn @click="retry(item.info)">
+              Retry
+            </v-btn>
+          </v-col>
+          <v-col class="pl-3" cols="auto">
+            <v-btn class="ignore-btn outlined" outlined @click="clearError(item.info)">
+              Ignore
+            </v-btn>
+          </v-col>
+        </v-row>
+      </tr>
+      <tr v-else :class="isExpanded ? 'no-hover' : ''" :colspan="headers.length" @click="expand(!isExpanded)">
+        <v-row no-gutters class="conflict-row py-5 px-4 border-top">
+          <v-col cols="9">
+            <v-row no-gutters>
+              <v-col cols="auto" class="nudge-down">
+                <v-icon :color="item.iconColor">{{ item.icon }}</v-icon>
+              </v-col>
+              <v-col cols="auto" class="table-text pl-3 pt-1">
+                <span v-html="item.problem"/>
+                <b v-for="(word, index) in item.words" :key=index>
```

Comment:
Another instance where the key index might spit out a complaint if it's not unique, not major though!

Reply:
yeah I'll update it, otherwise we might run into it later